llvm/unittests/Object/SymbolicFileTest.cpp

```diff
@@ -38,3 +40,24 @@ TEST(Object, DataRefImplOstream) {
 
   EXPECT_EQ(Expected, s);
 }
+
+struct ProxyContent {
+  unsigned Index = 0;
+  ProxyContent(unsigned Index) : Index(Index) {};
+  void moveNext() { Index++; }
```

Comment:
```suggestion
  void moveNext() { ++Index; }
```
see https://llvm.org/docs/CodingStandards.html#prefer-preincrement

Reply:
done